numpy/lib/shape_base.py

```diff
@@ -85,11 +85,9 @@ def apply_along_axis(func1d, axis, arr, *args, **kwargs):
     array([[[1, 0, 0],
             [0, 2, 0],
             [0, 0, 3]],
-
            [[4, 0, 0],
             [0, 5, 0],
             [0, 0, 6]],
-
```

Comment:
These lines are part of the correct repr. I wonder if there's some way we can keep them without messing up the formatting

Reply:
Probably with [`<BLANKLINE>`](https://docs.python.org/2/library/doctest.html#how-are-docstring-examples-recognized) but the other docstrings also just "omit" the blank lines in 3D arrays and I tend to prefer minimally invasive changes and adopting already existing conventions.

Reply:
Not sure how sphinx would handle a `<BLANKLINE>`, and it would make the contents of `help(...)` a little harder to read. 

Still, you're right - this is consistent with how we've done it before, and it's clearly better than doing nothing.